semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -176,6 +179,8 @@ def validate_values(self):
 
 
 class ResourceSemanticConvention(HasAttributes, BaseSemanticConvention):
+    TYPE_VALUE = "resource"
```

Comment:
The name TYPE_VALUE is unclear to me, maybe GROUP_TYPE_NAME (a group of this type has this name). Although that also sounds confusing

Reply:
:+1: